tests/mechanic/launcher_test.py

```diff
@@ -76,11 +77,15 @@ def info(self, *args, **kwargs):
 
 
 class ExternalLauncherTests(TestCase):
+    test_host = config_helper.TargetHosts('127.0.0.1:9200,10.17.0.5:19200')
```

Comment:
Nit: So far I used double-quotes for strings across the code-base and I think we should stick just to one representation.

Reply:
All :+1:  for standardization. I like the "safety" of no-interpolation with the single quote, but the code should be standardized. Will address.